tests/testthat/test_execution.R

```diff
@@ -76,56 +78,56 @@ test_that("Testing Execution Results Values", {
-    unlink(file.path(workingDir, file = "workflow2_output.rds"))
-    
+    expect_true(!any(grepl('_inputs.rds', list.files(outputDir))))
+    unlink(file.path(outputDir, file = "workflow2_output.rds"))
+
     capture.output(Execute(workflow2
                            , argsContainer = list(a = 4
                                                   ,b = -1)
-                           , moduleInputsSaveDir = workingDir
+                           , moduleInputsSaveDir = outputDir
                            , mode = PARALLEL_MODE))
     expect_equal({
-        readRDS(file.path(workingDir, "workflow2_output.rds"))
+        readRDS(file.path(outputDir, "workflow2_output.rds"))
     }, 3)
-    inputsRDSFiles = list.files(workingDir)
+    inputsRDSFiles = list.files(outputDir)
     expect_true('module1_2_inputs.rds' %in% inputsRDSFiles)
-    unserializedInputs <- readRDS(file.path(workingDir, file = "module1_2_inputs.rds"))
+    unserializedInputs <- readRDS(file.path(outputDir, file = "module1_2_inputs.rds"))
     expect_equal(class(unserializedInputs), 'list')
     expect_equal(length(unserializedInputs), 2)
     expect_equal(unserializedInputs$a, 4)
     expect_equal(unserializedInputs$b, -1)
-    unlink(file.path(workingDir, file = "workflow2_output.rds"))
-    unlink(file.path(workingDir, file = "module1_2_inputs.rds"))
-    
+    unlink(file.path(outputDir, file = "workflow2_output.rds"))
+    unlink(file.path(outputDir, file = "module1_2_inputs.rds"))
+
     capture.output(Execute(workflow2
                            , argsContainer = list(a = 4
                                                   ,b = -1)
-                           , moduleInputsSaveDir = file.path(workingDir,'does_not_exist')
+                           , moduleInputsSaveDir = file.path(outputDir,'does_not_exist')
                            , mode = PARALLEL_MODE))
-    inputsRDSFiles = list.files(file.path(workingDir, 'does_not_exist'))
+    inputsRDSFiles = list.files(file.path(outputDir, 'does_not_exist'))
     expect_true('module1_2_inputs.rds' %in% inputsRDSFiles)
-    unlink(file.path(workingDir, file = "workflow2_output.rds"))
-    unlink(file.path(workingDir, 'does_not_exist', file = "module1_2_inputs.rds"))
-    unlink(file.path(workingDir, 'does_not_exist'))
+    unlink(file.path(outputDir, file = "workflow2_output.rds"))
+    unlink(file.path(outputDir, 'does_not_exist', file = "module1_2_inputs.rds"))
+    unlink(file.path(outputDir, 'does_not_exist'))
 })
 
 #########################
 # CLEAN UP OUTPUT FILES.
 #########################
 # Rm output directory.
-outputDirs <- list.dirs(file.path(workingDir), recursive = FALSE)
+outputDirs <- list.dirs(file.path(outputDir), recursive = FALSE)
 unlink(outputDirs[grepl("output", outputDirs)], recursive = TRUE)
 
 # Rm misc data files.
-outputFiles <- list.files(workingDir, full.names = TRUE)
+outputFiles <- list.files(outputDir, full.names = TRUE)
```

Comment:
is there a reason you don't explicitly remove the `outputDir` itself?

Reply:
oh good catch. In the _original_ code, there was a pre-supplied output directory (not `tempdir()`) to be reused by all tests. Now that I'm using `tempdir()`, `updraft`'s tests probably don't need this output-checking code at all!

![image](https://user-images.githubusercontent.com/7608904/55296106-d56a4c00-53da-11e9-9d44-1cfe6f5d0c5d.png)


IMO that's a big enough contribution for you to document it in a separate issue and make it not a part of this PR (since it probably involves looking at all of this repo's tests)

Reply:
sounds good - thanks!